docs/specification.md

```diff
@@ -442,13 +445,15 @@ Updates or creates a new [position](#positions) in the given range.
   fails with `high_tokens_err` error code.
 - A user can set `delta_liquidity` to `0` on an existing position to simply retrieve
   any uncollected fees.
+- This fails with `price_out_of_bounds_err` when a tick index out of `[-1048575; 1048575]`
```

Comment:
Hmm it might not necessarily fail with `price_out_of_bounds_err`.

If both `lower_tick_index` and `lower_tick_witness` are lower than -1048575, then it'll fail with `tick_not_exist_err`.
If `lower_tick_index` is lower than -1048575, but `lower_tick_witness` is in range, then it'll fail with `invalid_witness_err`.

I think here we can simply say that the tick indices must in the range `[-1048575; 1048575]` (without mentioning which error will be thrown).